setup.py

```diff
@@ -2,7 +2,6 @@
 
 from distutils.core import setup
 from catkin_pkg.python_setup import generate_distutils_setup
-import enway_msgs.msg
 
```

Comment:
Was this by any chance an accidental delete?

Reply:
No actually, yesterday  I tested it without this import and it worked fine... So I deleted it...